inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp

```diff
@@ -721,70 +641,75 @@ struct jit_uni_normalize_kernel_f32 : public jit_uni_normalize_kernel, public ji
     }
 };
 
-MKLDNNNormalizeNode::MKLDNNNormalizeNode(const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache)
-        : MKLDNNNode(layer, eng, cache), src_data_size(0lu), dst_data_size(0lu), weights_data_size(0lu),
-        input_prec(Precision::UNSPECIFIED), output_prec(Precision::UNSPECIFIED), weights_prec(Precision::UNSPECIFIED) {}
+MKLDNNNormalizeNode::MKLDNNNormalizeNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
+        MKLDNNNode(op, eng, cache), src_data_size(0lu), dst_data_size(0lu), input_prec(Precision::UNSPECIFIED), output_prec(Precision::UNSPECIFIED) {
+    std::string errorMessage;
+    if (isSupportedOperation(op, errorMessage)) {
+        eps = std::dynamic_pointer_cast<const ngraph::op::v0::NormalizeL2>(op)->get_eps();
+        across_spatial = ngraph::shape_size(op->get_input_shape(AXES)) != 1;
+    } else {
+        THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED) << errorMessage;
+    }
+}
+
+bool MKLDNNNormalizeNode::isSupportedAxes(const std::vector<size_t> &axes, const ngraph::Shape &dataDims) {
```

Comment:
Looks like it shouldn't be encapsulated into MKLDNNNormalizeNode. Maybe the better to place it inside MKLDNNNormalizeNode::isSupportedOperation as lambda

Reply:
converted to lambda